.env.sample

```
@@ -33,6 +33,10 @@ OTEL_SERVICE_NAME=cex-broker
 # Broker → forwarder
 CEX_BROKER_ARCHIVE_FORWARDER_URL=http://localhost:8090/archive
 CEX_BROKER_DEPLOYMENT_ID=local-dev
+# OHLCV collector: required JSON array config and reconnect bootstrap coverage.
+CEX_BROKER_OHLCV_COLLECTOR_CONFIG=./ohlcv-subscriptions.json
+# 1000 one-minute bars cover roughly 16 hours on every (re)subscription.
+CEX_BROKER_OHLCV_ARCHIVE_BOOTSTRAP_LIMIT=1000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CEX_BROKER_OHLCV_ARCHIVE_BOOTSTRAP_LIMIT key should go before the CEX_BROKER_OHLCV_COLLECTOR_CONFIG key

(UnorderedKey)